sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverException.java

```diff
@@ -13,59 +11,33 @@
  *
  * @see ServiceBusErrorSource
  */
-public final class ServiceBusReceiverException extends Exception {
+public final class ServiceBusReceiverException extends AmqpException {
     private final transient ServiceBusErrorSource errorSource;
-    private final AmqpException amqpException;
 
     /**
      * @param amqpException for the error happened.
      * @param errorSource indicating which api caused the error.
      */
     ServiceBusReceiverException(AmqpException amqpException, ServiceBusErrorSource errorSource) {
```

Comment:
Should we have a constructor that just takes `Exception`? Then in the map all the receive errors that aren't AMQP exceptions to that. 
ServiceBusReceiverAsyncClient. L1198